### ISSUE TITLE
Train/Val accuracy

### ISSUE THREAD

**Issue by sirisha-8** [amaiya/ktrain#248]
@amaiya 

Hi,
I am new to keras, found ktrain easy to use .Right now when i am training NER related task using biobert model.I want to know the train /val accuracy. this info is not available in history .When i tried printing history keys i got the following output. 

`dict_keys(['loss', 'val_loss', 'lr'])`

Could you please let me know how to plot train/val accuracy.Will be very useful to my project.

Thanks in Advance

**Comment by amaiya** [comment]
You can obtain training/validation/test performance by invoking `learner.evaluate` (or `learner.validate`), as shown in the example notebooks.  By default, it will use the validation set, but you can supply a preprocessed version of the training or test set, as well.

By default, NER models in *ktrain* are not currently compiled with `metrics`, since standard classification accuracy isn't well-suited to measure accuracy of NER models.  The `learner.evaluate` and `learner.validate` methods compute the final F1 score (and precision and recall) using the [seqeval](https://pypi.org/project/seqeval/) package.  

If you want, you can recompile the model using whatever metric you want in which case it will computed at the end of each epoch and also be included in History. For instance, you can recompile model to use `seqeval.metrics.accuracy_score` prior to training:
```python
        >>> from seqeval.metrics import accuracy_score
        >>> y_true = [['O', 'O', 'O', 'B-MISC', 'I-MISC', 'I-MISC', 'O'], ['B-PER', 'I-PER', 'O']]
        >>> y_pred = [['O', 'O', 'B-MISC', 'I-MISC', 'I-MISC', 'I-MISC', 'O'], ['B-PER', 'I-PER', 'O']]
        >>> accuracy_score(y_true, y_pred)
        0.80
```
